lib/src/main/java/io/ably/lib/http/HttpUtils.java

```diff
@@ -71,4 +109,24 @@ public static String encodeURIComponent(String input) {
 		} catch (UnsupportedEncodingException e) {}
 		return null;
 	}
+
+	public static BodyHandler<JsonElement> jsonArrayResponseHandler = new BodyHandler<JsonElement>() {
+		@Override
+		public JsonElement[] handleResponseBody(String contentType, byte[] body) throws AblyException {
+			if(!"application/json".equals(contentType)) {
+				return null;
```

Comment:
Silent failure?  I am not suggesting we raise an exception necessarily, but would it be at least logging a warning?